src/core.js

```diff
@@ -1894,10 +1894,13 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
     let j;
 
     if (isDefined(settings.rows)) {
-      throw new Error('"rows" setting is no longer supported. do you mean startRows, minRows or maxRows?');
+      throw new Error('The "rows" setting is no longer supported. do you mean startRows, minRows or maxRows?');
```

Comment:
👍 for noticing that. I've noticed one too 🙈 

```suggestion
      throw new Error('The "rows" setting is no longer supported. Do you mean startRows, minRows or maxRows?');
```